style.css

```diff
@@ -71,5 +75,226 @@ ul {
   justify-content: flex-start;
   align-items: flex-start;
   position: relative;
-  right: 45px;
+  right: 40px;
+}
+
+.tonic2 {
+  display: flex;
+  flex-direction: column;
+  min-height: 718px;
+  max-width: 300px;
+  left: 24px;
+  top: 114px;
+  border-radius: 16px;
+  padding: 16px;
+}
+
+.tnc {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin-bottom: 0.5px;
+}
+
+.can {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  position: relative;
+  top: 2px;
+}
+
+.li2 {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.ul2 .can {
+  list-style-type: none;
+}
+
+.ul2 {
+  display: flex;
+  justify-content: space-around;
+  position: relative;
+  right: 55px;
+  margin-bottom: 0.5px;
+}
+
+.tonic2 p {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.ul3 {
+  display: flex;
+  list-style-type: none;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  color: #6070ff;
+  justify-content: space-around;
+  position: relative;
+  right: 50px;
+  width: fit-content;
+}
+
+.ul3 li {
+  background-color: #ebebff;
+  border-radius: 8px;
+  margin: 0 8px;
+  padding: 8px 24px;
+}
+
+.see {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  padding: 10px;
+  border: 1px solid #6070ff;
+  position: relative;
+  top: 30px;
+  text-decoration: none;
+  border-radius: 8px;
+  width: fit-content;
+  display: block;
+}
+
+.see:hover {
+  background-color: #6070fd;
+  color: white;
+}
+
+a {
+  color: #396df2;
+}
+
+.ul2 li::marker {
+  font-size: 1.5em;
+}
+
+.outro {
+  min-height: 718px;
```

Comment:
- [OPTIONAL] Kindly ensure to fix this issue by the previous reviewer:

 Please notice that using the `min-height` property means that the minimal height could this section ever get is its value (718px) and this might be too much in some situations.
_Kindly work on fixing this issue as requested earlier by the previous reviewer, and you'll be good to go._
**Hint**
Using `margin` to widen the space between elements or sections is a better way than declaring a fixed height for them.
```
.section {
  margin: 1rem 0; // 1rem for top and bottom margins, 0rem for right and left margins
}
```

> Please fix the spacings on all similar situations through the website 🤗

_Originally posted by @MahmoudBakr23 in https://github.com/isaka-lumato/portfolio/pull/2#r650362197_